apps/web/src/pages/api/userScore.ts

```diff
@@ -1,69 +1,87 @@
-import { AccountProfile } from 'types/Score';
+import { NextApiRequest, NextApiResponse } from 'next';
+import { UserScore, Tweet } from 'types/Score';
+import { config } from '../../config';
 import { userTweets } from '../../modules/twitter/twitterFollowersGet';
 
-export const userScore = async ({
-  account: { providerAccountId, access_token },
-  profile: {
-    data: { username },
-  },
-}: AccountProfile) => {
-  if (!access_token) {
-    return 0;
-  }
-
-  const result = await userTweets(providerAccountId, access_token);
-
-  if (!result.data) {
-    return 0;
-  }
-
-  const { data } = result;
-
-  const currentUser = username;
-  interface PublicMetrics {
-    public_metrics: {
-      retweet_count: number;
-      reply_count: number;
-      like_count: number;
-      quote_count: number;
-    };
-  }
-
-  const scoreDetail = {
+export default async function handler(
```

Comment:
userScoreHandler is a better name

Reply:
Do I change the handler of tweets.ts to tweetsHandler ?
![image](https://user-images.githubusercontent.com/5343791/159579215-df788bc0-d1f0-4dbd-ba75-d4a050bdb309.png)

Reply:
we need to normalize